blocked-edges/4.15.0-rc.0-OVNIPsecConnectivityForANewCase.yaml

```diff
@@ -0,0 +1,20 @@
+to: 4.15.0-rc.0
```

Comment:
nit: I don't see RCs called out explicitly in [CORENET-5630](https://issues.redhat.com/browse/CORENET-5630), and the `PreRelease` risk has these protected already, so I think we can skip them and stick with GA releases for this new risk, instead of trying to figure out when in the 4.15 prerelease cycle exactly this regression snuck in.

Reply:
Removed RCs.
And rename to `name: OVNIPsecPausedMCPConnectivityForHostPods`.